src/main/java/io/github/antishake/CircularBuffer.java

```diff
@@ -11,43 +10,48 @@
         -will maintain write pointer when updating with the latest values
  */
 public class CircularBuffer {
-    public int read_pointer,write_pointer=0;
-    public int size_buffer =201;
-    double[] circularbuffer =new double[size_buffer];
-    boolean buffer_full_once=false;
-    public boolean isEmpty(){
-        return read_pointer==write_pointer;
+  private int readPointer, writePointer = 0;
+  private int sizeBuffer = 201;
```

Comment:
Please do not hard code the value of the size. Make a constructor so that whoever needs the circular buffer data structure can provide the size whatever they require through the constructor.